manual/cops_bundler.md

```diff
@@ -6,11 +6,36 @@ Enabled by default | Supports autocorrection
 --- | ---
 Enabled | No
 
-This cop checks for duplicate gem entries in Gemfiles. Bundler currently
-only prints a warning (unless there is a requirements conflict).
+A Gem's requirements should be listed only once in a Gemfile.
```

Comment:
@Drenmi I took a stab at a really simple way to enforce this check: https://github.com/bbatsov/rubocop/pull/3706